### PR TITLE
Done

### PR DESCRIPTION
**Pull request by meganeloise**

### CODE REVIEW

File: students/megan-eloise.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  </head>
+  <body>
+<div id="container4">
+  <div id="container3">
+    <div id="container2">
+      <div id="container1">
+        <div id="col1">
+          <h1>Hi, I'm Megan Eloise</h1>
+          <p>You can find my projects <a href="http://www.meganeloise.me/">here.</a></p>
+
```
Comment (bhollan) [inline students/megan-eloise.html]: Hi there!  I think you're missing a lot of closing tags and other formatting.  Take a look at another student's profile file and you can use that as a reference to see what you're missing.

File: css/styles.css [hunk 2]
```diff
@@ -19,20 +19,22 @@ body {
 html, body {
   width: 100vw;
   min-height: 100vh;
+  padding: 40px;
```
Comment (bhollan) [inline css/styles.css]: This line should not be here.  You should only add your two rules for your name's `cover` and `card` in this file.

File: css/styles.css [hunk 3]
```diff
@@ -19,20 +19,22 @@ body {
...
+
 }
 
 h1, h2, h3, h4, h5, h6 {
   margin: 0;
 }
 
 a {
-  color: #fff;
+  color: fff;
```
Comment (bhollan) [inline css/styles.css]: The `#` is there to denote it as a Hex value.  Color as `fff` doesn't make sense.

File: css/styles.css [hunk 4]
```diff
@@ -19,20 +19,22 @@ body {
...
   text-decoration: none;
 }
 
 a:visited {
   text-decoration: none;
-  color: #fff;
+  color: #red;
```
Comment (bhollan) [inline css/styles.css]: Here, it's the opposite.  Having `red` by itself would make sense, but having `#red` does not.  You can look up different ways to represent color in CSS and read all about it.

**Comment by bhollan** [comment]
CC: @aturkewi

**Comment by ivalentine** [comment]
Hi @meganeloise! I'm going to close this pull request. If you would like to make those changes and have them merged, just re-open this pull request and let us know.

Cheers!

cc @aturkewi

**Pull request closed without merging**